devel/astyle/Portfile

```diff
@@ -18,15 +19,13 @@ long_description    Artistic Style is a source code indenter, source code format
                     programming languages.
 
 homepage            http://astyle.sourceforge.net
```

Comment:
Please also set the homepage URL to HTTPS

Reply:
Done

Reply:
Thanks